recipes-sota/rac/rac_git.bb

```diff
@@ -14,7 +14,7 @@ SRC_URI = " \
 
 SRCREV_FORMAT = "rac_tough"
 
-SRCREV_rac = "2b0f1189e344f8205b0df7e54e114666da31732e"
+SRCREV_rac = "15845bcaad19b9cbb79816b84184547df559e6b5"
```

Comment:
The recipe here did not need any updating? With Rust-based recipes sometimes the entire recipe needs to be updated when the hash gets bumped due to changed/updated dependencies.